### ISSUE TITLE
undefined reference to dlclose on arch, in crosstool-ng

### ISSUE THREAD

**Issue by stebulus** [stebulus/egg-linux#26]
Building b34d253 on a fresh Arch Linux instance in EC2:
```
...
  LD     'conf'
  LD     'mconf'
  LD     'nconf'
/mnt/egg-linux/x/build-toolchain/bin/ld: /mnt/egg-linux/x/ncurses/lib/../lib/libncurses.a(lib_mouse.o): in function `load_gpm_library':
lib_mouse.c:(.text+0x5a8): undefined reference to `dlopen'
/mnt/egg-linux/x/build-toolchain/bin/ld: lib_mouse.c:(.text+0x5c7): undefined reference to `dlsym'
/mnt/egg-linux/x/build-toolchain/bin/ld: lib_mouse.c:(.text+0x5e6): undefined reference to `dlsym'
/mnt/egg-linux/x/build-toolchain/bin/ld: lib_mouse.c:(.text+0x605): undefined reference to `dlsym'
/mnt/egg-linux/x/build-toolchain/bin/ld: lib_mouse.c:(.text+0x624): undefined reference to `dlsym'
/mnt/egg-linux/x/build-toolchain/bin/ld: lib_mouse.c:(.text+0x666): undefined reference to `dlclose'
/mnt/egg-linux/x/build-toolchain/bin/ld: /mnt/egg-linux/x/ncurses/lib/../lib/libncurses.a(lib_mouse.o): in function `enable_gpm_mouse':
lib_mouse.c:(.text+0x6c8): undefined reference to `dlclose'
collect2: error: ld returned 1 exit status
make[2]: *** [nconf] Error 1
make[1]: *** [build-lib-kconfig] Error 2
make: *** [build] Error 2
leaving behind: /mnt/tmp/crosstool-ng.Y1cQOLgC
```
Presumably due to b34d253 itself.

**Comment by stebulus** [comment]
The annoying thing about this problem, and the problem on Slackware 14.2 that b34d253 was meant to solve, is that we don't even need the ncurses interface.

**Comment by stebulus** [comment]
Likely fix in dcd0105: just not building nconf.

**Comment by stebulus** [comment]
I opened #30 to track a less hacky solution.
